### PR TITLE
add user ssh key documentation

### PR DESCRIPTION
**Pull request by timogoebel**
This adds a small paragraph about the user ssh key feature added in 1.15.

### CODE REVIEW

**Comment by timogoebel** [comment]
cc: @ares

File: _includes/manuals/1.15/4.1.6_user_management.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+Foreman is all about hosts and users interacting with these hosts.
+
+#### SSH Keys
+
+Each Foreman user can have multiple SSH keys assigned when editing a user. These keys alone do not serve any purpose, but are available for use in provisioning templates and can be accessed via ENC data. They provide an easy way to manage users and login ssh keys on hosts without the need for LDAP.
+If you want users to be able to login to a host using the data provided in Foreman, you need to include the `create_users` snippet in your provisioning template. There is a [puppet module](https://github.com/ekohl/puppet-foreman_simple_user) available to keep user data in sync with Foreman and your hosts.
```
Comment (ares) [inline _includes/manuals/1.15/4.1.6_user_management.md:7]: @ekohl how much is the puppet module supported by you? Would you be up for moving under theforeman org?

**Comment by ares** [comment]
LGTM :+1: , waiting on answer from Ewoud

File: _includes/manuals/1.15/4.1.6_user_management.md (hunk 1 above)
Comment (ekohl) [reply _includes/manuals/1.15/4.1.6_user_management.md:7]: Definitely, after a review from others and modulesync then we can move it to theforeman org. After publishing it to the forge it should also be easier to use.

**Review by ekohl — APPROVED** [review]
Looks good. I'm not sure if we should wait to move the puppet module or just publish this now.

File: _includes/manuals/1.15/4.1.6_user_management.md (hunk 1 above)
Comment (timogoebel) [reply _includes/manuals/1.15/4.1.6_user_management.md:7]: As I read the code when you visited Germany, I'd consider the review done. :-)

**Comment by timogoebel** [comment]
> I'm not sure if we should wait to move the puppet module or just publish this now.

Let's publish this now. We can easily improve this or change the link later.

File: _includes/manuals/1.15/4.1.6_user_management.md (hunk 1 above)
Comment (ekohl) [reply _includes/manuals/1.15/4.1.6_user_management.md:7]: @mmoll I'd like to get the module moved and released on the forge but might not have time for it until next week. Think we could work on that then?

Comment (mmoll) [reply _includes/manuals/1.15/4.1.6_user_management.md:7]: I can't help with the move to the github org (@domcleal? @ohadlevy?), but with modulesync and forge things.

**Comment by ares** [comment]
Sure, I was curious if that's somewhere on the roadmap :-) Thanks @timogoebel, merging!

**Pull request merged**